spring-vault-core/src/test/java/org/springframework/vault/util/VaultInitializer.java

```diff
@@ -96,7 +96,7 @@ public void initialize() {
 
 			if (this.prepareVault.getVersion()
 					.isGreaterThanOrEqualTo(VERSIONING_INTRODUCED_WITH)) {
-				this.prepareVault.disableGenericVersioning();
+				//this.prepareVault.disableGenericVersioning();
```

Comment:
I know this is not the right fix, but I was just wondering why using `/versioned` path and unmounting the already provided `/secret`. In real world settings, `/secret` is the default right?

Reply:
This is true if you look at it from a most recent version perspective. We're running our test against various Vault versions that do not contain the KV version 2 backend so we're creating a stable setup. Also, newer Vault versions were used to mount the versioned backend at `/secret` so we need to revert this change.